mirage/seed_image/catalog_seed_image.py

```diff
@@ -3076,22 +3080,27 @@ def make_galaxy_image(self, file):
 
             # First create the galaxy
             stamp = self.create_galaxy(entry['radius'], entry['ellipticity'], entry['sersic_index'],
-                                       xposang*np.pi/180., entry['counts_per_frame_e'])
+                                       xposang*np.pi/180., entry['countrate_e/s'])
 
             # If the stamp image is smaller than the PSF in either
             # dimension, embed the stamp in an array that matches
             # the psf size. This is so the upcoming convolution will
             # produce an output that includes the wings of the PSF
             galdims = stamp.shape
 
-            # *******OR SHOULD WE ALWAYS CONVOLVE WITH A LARGER PSF? IF PSF IS SMALL,
-            # THIS WILL ARTIFICIALLY PUSH SIGNAL TOWARDS THE CORE OF THE GALAXY, I THINK.
-            #psf_dimensions = self.find_psf_size(entry['countrate_e/s'])
-            #psf_shape = np.array([psf_dimensions, psf_dimensions])
-
+            #print('Uncomment the lines below to return to the case where only the')
+            #print('PSF core is used for convolution')
             psf_dimensions = np.array(self.psf_library.data.shape[-2:])
             psf_shape = np.array((psf_dimensions / self.psf_library_oversamp) -
                                  self.params['simSignals']['gridded_psf_library_row_padding']).astype(np.int)
+
+
+            # Find the PSF size to use based on the countrate
+            #print('\n\nTEST OF INCLUDING WINGS IN PSF TO CONVOLVE WITH GALAXY\n\n')
+            #psf_x_dim = self.find_psf_size(entry['countrate_e/s'])
+            #psf_shape = [psf_x_dim, psf_x_dim]
+            #print('\n\nTEST OF INCLUDING WINGS IN PSF TO CONVOLVE WITH GALAXY\n\n')
```

Comment:
Should these commented-out lines be removed from this PR, or is it intentional to keep them?

Reply:
I tend to err on the side of leaving anything that may be useful in the future (for example, if we changed our minds and decided that it would in fact be better to use the full PSF for convolution) in the code. But it does make things more messy. I'll pull this out.